woocommerce/woocommerce
# CYS: Revisit sidebar layout

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:






| Before | After |
|--------|--------|
|  <video src="https://github.com/woocommerce/woocommerce/assets/4463174/ff5f1e88-41fa-4d47-9a11-7485b3de97f0" /> |<video src="https://github.com/woocommerce/woocommerce/assets/4463174/04752a9b-2dba-4b7f-9b0d-e8a8dd394d5f" /> | 







Closes #48783.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that you have the last version of Gutenberg enabled
2. Make sure you have the WooCommerce beta tester plugin installed
3. Head over to Tools > WCA Test Helper > Features
4. Make sure you see the `pattern-toolkit-full-composability` on the list.
5. Enable it.
6. Head over to WooCommerce > Home > Customize your store.
7. Click on Design Your Homepage.
8. Click on a category.
9. Ensure that after clicking the category, it is still highlighted.
10. Hover patterns in the second sidebar and ensure that the border looks like the video in the "after" column



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

CYS: Revisit sidebar layout.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
